.test-infra/dataproc/init-actions/beam.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+#    Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I know that [Beam init action](https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/blob/master/beam/beam.sh) exists in Dataproc repo too, but IMO it didn't fit our needs. I provided a version that in my opinion is very different. It's got a Beam license header. Is this ok?

Reply:
That seems fine to me.